servers/mekbuda/default.nix

```diff
@@ -0,0 +1,29 @@
+{ modulesPath, inputs, config, lib, ... }:
+{
+  imports = [
+    "${modulesPath}/virtualisation/amazon-image.nix"
```

Comment:
I would also add `systemd.services.amazon-init.enable = false`, to be sure that amazon-init script does not run nixos-rebuild on boot